### ISSUE TITLE
Using with nvim-compe gives error when loading vscode snippets with multiple options. [ERROR] attempt to concatenate a table value

### ISSUE THREAD

**Issue by rafamadriz** [L3MON4D3/LuaSnip#63]
Error output: 

`Error executing vim.schedule lua callback: .../pack/packer/start/nvim-compe/lua/compe_luasnip/init.lua:27: attempt to concatenate a table value`

Minimal init.lua to reproduce:

```lua
-- This file is a minimal `init.lua` example for debuggin plugins.
-- Open with `nvim -u debug.lua`

local fn = vim.fn
local install_path = "/tmp/nvim/site/pack/packer/start/packer.nvim"

vim.cmd([[set packpath=/tmp/nvim/site]])
local function load_plugins()
	local use = require("packer").use
	require("packer").startup({
		function()
			use("wbthomason/packer.nvim")
			-- Put plugins to install here:
			use("hrsh7th/nvim-compe")
			use("L3MON4D3/LuaSnip")
			use("rafamadriz/friendly-snippets")
		end,
		config = { package_root = "/tmp/nvim/site/pack" },
	})
end

_G.load_config = function()
	vim.o.completeopt = "menuone,noselect"
	require("compe").setup({
		enabled = true,
		preselect = 'enable',
		autocomplete = true,
		source = {
			luasnip = true,
		},
	})
	local function prequire(...)
		local status, lib = pcall(require, ...)
		if status then
			return lib
		end
		return nil
	end

	local luasnip = prequire("luasnip")

	local t = function(str)
		return vim.api.nvim_replace_termcodes(str, true, true, true)
	end

	local check_back_space = function()
		local col = vim.fn.col(".") - 1
		if col == 0 or vim.fn.getline("."):sub(col, col):match("%s") then
			return true
		else
			return false
		end
	end

	_G.tab_complete = function()
		if vim.fn.pumvisible() == 1 then
			return t("<C-n>")
		elseif luasnip and luasnip.expand_or_jumpable() then
			return t("<Plug>luasnip-expand-or-jump")
		elseif check_back_space() then
			return t("<Tab>")
		else
			return vim.fn["compe#complete"]()
		end
	end
	_G.s_tab_complete = function()
		if vim.fn.pumvisible() == 1 then
			return t("<C-p>")
		elseif luasnip and luasnip.jumpable(-1) then
			return t("<Plug>luasnip-jump-prev")
		else
			return t("<S-Tab>")
		end
	end

	local map = vim.api.nvim_set_keymap
	vim.cmd("inoremap <silent><expr> <CR>      compe#confirm('<CR>')")
	map("i", "<Tab>", "v:lua._G.tab_complete()", { expr = true })
	map("s", "<Tab>", "v:lua._G.s_tab_complete()", { expr = true })
	map("i", "<S-Tab>", "v:lua.MUtils.s_tab_complete()", { expr = true })
	map("s", "<S-Tab>", "v:lua.MUtils.s_tab_complete()", { expr = true })
	require("luasnip.loaders.from_vscode").load()
end

if fn.isdirectory(install_path) == 0 then
	-- install packer
	fn.system({ "git", "clone", "https://github.com/wbthomason/packer.nvim", install_path })
	load_plugins()
	require("packer").sync()
	vim.cmd("autocmd User PackerComplete ++once lua load_config()")
else
	load_plugins()
	_G.load_config()
end
```

Steps:

1. Open minimal init.lua example.
2. Edit a file and trigger a snippet that has multiple options.

Here a video.

In this case I'm trying to use the `shebang` snippet which has multiple options, (see [here](https://github.com/rafamadriz/friendly-snippets/blob/21b6ffd6d79cc16272c889e17ae7db3eb2cf8199/snippets/shell.json#L4))

https://user-images.githubusercontent.com/67771985/124820754-671f7700-df5d-11eb-8712-9264550d942d.mp4

**Comment by L3MON4D3** [comment]
Ah, I think the issue isn't that the snippet has multiple options, its that the multiline description isn't considered when parsing the `.json`.
I'll look into fixing that, should be rather straighforward.

**Comment by L3MON4D3** [comment]
This should be resolved by hrsh7th/nvim-compe#457.

**Comment by L3MON4D3** [comment]
I'm going to close this as the PR has been merged into compe.
